cmd/subnetcmd/upgradecmd/print_upgrade.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package upgradecmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanchego/utils/storage"
+	"github.com/spf13/cobra"
+)
+
+// avalanche subnet upgrade import
+func newUpgradePrintCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "print [subnetName]",
+		Short: "Print the upgrade.json file content",
+		Long:  `Print the upgrade.json file content`,
+		RunE:  upgradePrintCmd,
+		Args:  cobra.ExactArgs(1),
+	}
+
+	return cmd
+}
+
+func upgradePrintCmd(cmd *cobra.Command, args []string) error {
+	subnetName := args[0]
+	if !app.GenesisExists(subnetName) {
+		ux.Logger.PrintToUser("The provided subnet name %q does not exist", subnetName)
+		return nil
+	}
+
+	subnetPath := filepath.Join(app.GetUpgradeFilesDir(), subnetName)
```

Comment:
Might want to make this whole filepath constructing process part of `app` like I mentioned in the other PR.

Reply:
Yep, just reusing the `upgrades.ReadFile` func from #333